docs/materials/htc/part2-ex1-files.md

```diff
@@ -85,13 +85,13 @@ for word in sorted(words.keys()):
 1.  Download the input file for the script (263K lines, ~1.4 MB) and save it in your submit directory:
 
         :::console
-        username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/osgschool19/mon-2.1-words.txt
+        username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/osgschool20/intro-2.1-words.txt
```

Comment:
This is going to conflict with PR 17… Anyway, has anyone verified that this directory exists for this year? Did this command work?

Reply:
Yes, I copied it inside `/squid/osgschool20` and http://proxy.chtc.wisc.edu/SQUID/osgschool20/intro-2.1-words.txt works